src/composables/useClient.ts

```diff
@@ -86,41 +79,31 @@ export function useClient() {
         app: DEFINED_APP,
         reason: payload.reason
       });
-      return receipt;
     } else if (type === 'delete-proposal') {
-      const receipt = await client.cancelProposal(
-        auth.web3,
-        web3.value.account,
-        {
-          space: space.id,
-          proposal: payload.proposal.id
-        }
-      );
-      return receipt;
+      return await client.cancelProposal(auth.web3, web3.value.account, {
+        space: space.id,
+        proposal: payload.proposal.id
+      });
     } else if (type === 'settings') {
-      const receipt = await client.space(auth.web3, web3.value.account, {
+      return await client.space(auth.web3, web3.value.account, {
         space: space.id,
         settings: JSON.stringify(payload)
       });
-      return receipt;
     } else if (type === 'delete-space') {
-      const receipt = await client.deleteSpace(auth.web3, web3.value.account, {
+      return await client.deleteSpace(auth.web3, web3.value.account, {
         space: space.id
       });
-      return receipt;
     } else if (type === 'set-statement') {
-      const receipt = await client.statement(auth.web3, web3.value.account, {
+      return await client.statement(auth.web3, web3.value.account, {
         space: space.id,
         about: payload.about,
         statement: payload.statement
       });
-      return receipt;
     } else if (type === 'flag-proposal') {
-      const receipt = await client.flagProposal(auth.web3, web3.value.account, {
+      return await client.flagProposal(auth.web3, web3.value.account, {
```

Comment:
I think we could skip the `await` here

Reply:
we could but don't want to test it (v1 will deprecate anyway so) 😅 ok will try to remove